devnet-sdk/devstack/shim/sequencer.go

```diff
@@ -0,0 +1,59 @@
+package shim
+
+import (
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-service/apis"
+	"github.com/ethereum-optimism/optimism/op-service/client"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+)
+
+type SequencerConfig struct {
+	CommonConfig
+	ID                 stack.SequencerID
+	Client             client.RPC
+	L2SequencerClients map[eth.ChainID]client.RPC
```

Comment:
`SequencerConfig` has a set of pairs to concrete L2 sequencers, although `WithSequencer` currently hydrates the sequencer with value for a single L2 chain.